app/controllers/organisations_controller.rb

```diff
@@ -129,7 +133,9 @@ def self.build params
   private
 
   def set_organisation
-    @organisation = Organisation.friendly.find(params[:id]) rescue nil
+    @organisation = Organisation.friendly.find(params[:id])
+    rescue ActiveRecord::RecordNotFound
+      @organisation = nil
```

Comment:
the changes in the "set_organisation" method where an attempt to eliminate "codeclimate" issues stating, "avoid using rescue in its modifier form"